### PR TITLE
NEW Added tag to deployment references

### PR DESCRIPTION
**Pull request by micmania1**

### CODE REVIEW

**Comment by micmania1** [comment]
deploynaut-tags

File: code/model/jobs/DNDeployment.php [hunk 1]
```diff
@@ -166,6 +174,28 @@ public function getCommitMessage() {
...
+		if(!$this->SHA) {
+			return null;
+		}
+
+		$repo = $this->getRepository();
+		if($repo) {
+			$tags = $repo->getReferences()->resolveTags($this->SHA);
+			if(!empty($tags)) {
+				$field = Varchar::create('Tag', '255');
+				$field->setValue(end($tags)->getName());
```
Comment (mateusz) [inline code/model/jobs/DNDeployment.php]: This could get confusing if someone happens to have more than one tag

Comment (micmania1) [inline code/model/jobs/DNDeployment.php]: I could add getTags() too?

**Comment by micmania1** [comment]
With multiple tags:
deploynaut-tags-2

**Pull request merged**